configure.py

```diff
@@ -699,6 +699,7 @@ def find_headers(repodir, excluded_dirs):
                 'dht/i_partitioner.cc',
                 'dht/token.cc',
                 'dht/murmur3_partitioner.cc',
+                'dht/cdc_partitioner.cc',
```

Comment:
To me it makes more sense to put cdc_partitioner in cdc/. Yes, it's a partitioner, but it's a partitioner to be used *only* by CDC so will be nice to put it with the rest of the cdc-specific code. If you think it could be used outside CDC, then we should find a better name for it (maybe "identity_partitioner"?), not "cdc_partitioner".

Reply:
I will move it.

Reply:
Agree

Reply:
done.